### PR TITLE
Code for #50

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

**Comment by jlipps** [comment]
looks good; is it possible to just print out startup time in seconds to stdout, in lieu of the test code? so that at least some difference is detectable when running things?

**Comment by Jonahss** [comment]
Ah, I was just looking at my IDE which listed test runtimes. I'll add that.

File: java/src/test/java/Edition050_Faster_Android_Capabilities.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    @Test
+    public void testSkippingInstallOfDependencies() throws IOException {
+        DesiredCapabilities caps = new DesiredCapabilities();
+
+        caps.setCapability("platformName", "Android");
+        caps.setCapability("deviceName", "Android Emulator");
+        caps.setCapability("automationName", "UiAutomator2");
+
+        // App is already installed on device, so can be launched by Package name and Activity
```
Comment (vsadineni1) [inline java/src/test/java/Edition050_Faster_Android_Capabilities.java]: @Jonahss one curious question. In the 2nd test(testSkippingInstallOfDependencies) you mentioned that the app is already available. Do you mean that the app is loaded by the first test(testInstallingDependencies).
If that is the case, are we not assuming the sequence of execution of these tests?

Comment (Jonahss) [reply java/src/test/java/Edition050_Faster_Android_Capabilities.java]: We are totally assuming that the tests will be run in sequence and not in parallel. Is there a way I could make that more clear?

Should I combine them into a single test case? Or make one depend upon the other explicitly?

Comment (vsadineni1) [reply java/src/test/java/Edition050_Faster_Android_Capabilities.java]: Thanks for clarifying that. 
I feel like may be it is a good idea if we can make one dependent on another, that way we can grantee the execution sequence.

Comment (Jonahss) [reply java/src/test/java/Edition050_Faster_Android_Capabilities.java]: Good idea, thank you for the suggestion.

Comment (Jonahss) [reply java/src/test/java/Edition050_Faster_Android_Capabilities.java]: @vsadineni1 How would you suggest I make them dependent. I could put the first test as an @Before, but then it's not clear that it's a test itself.

Comment (vsadineni1) [reply java/src/test/java/Edition050_Faster_Android_Capabilities.java]: @Jonahss the below link shows an example using the annotation: @FixMethodOrder(MethodSorters.NAME_ASCENDING)
https://github.com/junit-team/junit4/wiki/test-execution-order

Comment (Jonahss) [reply java/src/test/java/Edition050_Faster_Android_Capabilities.java]: Oh that's perfect. Thanks.

**Pull request closed without merging**